.github/workflows/build.yml

```diff
@@ -2,9 +2,6 @@ name: build
 
 on:
   pull_request:
```

Comment:
rather than deleting -develop and -master, it'd be better to just add your feature branch. I'm not sure what the outcomes might be of the automated versioning script. It only does some bits for develop and master, but the whole script isn't wrapped in an if, so there might be some unpredictable results.

Reply:
I see what you mean. There is something below which says run the new-version if: ${{ github.event_name == 'push' }} . But you're right, we don't really know the affects of it are. I think rather than messing with it in this branch I'll create a separate ticket for it. For now I'll just revert it.